lib/ruleHelper.js

```diff
@@ -111,7 +111,7 @@ RuleHelper.prototype = {
             break;
         case "MemberExpression":
             methodName = node.property.name;
-            objectName = node.object.name;
+            objectName = node.object.name || this.context.getSource(node.object);
```

Comment:
*Maybe* this should be more explicit?
Not sure this is worth time thinking about. This will be checked with a regex after all.